_posts/en/posts/schnorr-sigagg.md

```diff
@@ -0,0 +1,84 @@
+---
+type: posts
+layout: post
+lang: en
+name: schnorr-sigagg
+id: en-schnorr-sigagg
+title: Technology roadmap: Schnorr signatures & signature aggregation
+permalink: /en/2017/03/22/schnorr-sigagg/
+version: 1
+excerpt: Status and explanation of Schnorr signatures and signature aggregation
+---
+
+## Schnorr Signatures
+
+The replacement of Bitcoin’s digital signature algorithm (ECSDA) for the more efficient Schnorr algorithm has long been at the top of the bucket list for many Bitcoin developers. A simple algorithm leveraging elliptic curves cryptography, Schnorr enables several improvements over the existing scheme all while preserving all of the features and security assumptions of ECSDA.
```

Comment:
> top of the bucket list for many Bitcoin developers.

Around here, a bucket list is a list of things you want to do before you "kick the bucket", that is---die.  Are we expecting a rash of developer fatalities?  If not, maybe "bucket" could be dropped from this sentence.

> elliptic curves cryptography

I think "curves" should be singular, [e.g.](https://en.wikipedia.org/wiki/Elliptic_curve_cryptography).

> Schnorr enables several improvements over the existing scheme all while preserving all of the features and security assumptions of ECSDA.

s/ECSDA/ECDSA/; in addition, I suggest this rephrase for clarity: "over the existing ECDSA scheme all while preserving all of its features and security assumptions."